docs/guides/breakings/v3.18.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: Guides.Breakings.V3_18
+title: V3.18
+---
+
+# V3.18
+
+Among the changes in V3.18 was components V2, while splendid, it did bring with it a few changes you should be aware of.
+
+- IMessage
+
+Messages components property has changed. it is now a MessageComponent type. Effectively what that means for Components V2 is that you can simply call Build() on an instance of ComponentBuilderV2 like so:
```

Comment:
This should mention that `IReadOnlyCollection<ActionRowComponent> Components` is now `IReadOnlyCollection<IMessageComponent> Components`
and that a patch for this would be doing a LINQ query `.OfType<ActionRowComponent>` on the collection.

Reply:
And it has nothing to do with component builder v2, as one is for sending components. this one is only for receiving